weathervane.pl

```diff
@@ -261,17 +335,18 @@ sub createComputeResource {
 $appender = Log::Log4perl::Appender->new(
 	"Log::Dispatch::File",
 	name     => "rootDebugFile",
-	filename => "$weathervaneHome/debug.log",
+	filename => "$tmpDir/debug.log",
 	mode     => "write",
 );
```

Comment:
Do you have any concern this might cause any adverse impacts? (performance, disk space, functional)

Reply:
I don't think that there would be any performance or function issues.  I'm actually not sure how large the debug output gets.  I'll try a run and then comment again.

Reply:
The debug log for a single run is about 750KB.  A little large but not prohibitive.  I say go ahead and set the default log level to DEBUG.  I will open a separate issue to review what is going to that log so that we can clean it up and reduce the size a bit.